src/structures/paging/mapper/mapped_page_table.rs

```diff
@@ -549,7 +549,7 @@ impl<'a, P: PageTableFrameMapping> Translate for MappedPageTable<'a, P> {
             Err(PageTableWalkError::MappedToHugePage) => {
                 let entry = &p3[addr.p3_index()];
                 let frame = PhysFrame::containing_address(entry.addr());
-                let offset = addr.as_u64() & 0o_777_777_7777;
+                let offset = addr.as_u64() & 0o7_777_777_777;
```

Comment:
I would prefer to keep the grouping as it represents the page table layout (12 bits page offset, 9 bits page table index per level). Without the grouping, it doesn't make much sense to use octal instead of hexadecimal for these numbers.

So I think it would be better to `allow(clippy::unusual_byte_groupings)` in this and in similar cases.

Reply:
Totally! I just could not wrap my head around why you chose this grouping, but this makes sense! 👍 

Changed.